rclrs/src/node.rs

```diff
@@ -360,6 +360,84 @@ impl Node {
         domain_id
     }
 
+    /// Creates a `ParameterBuilder` that can be used to set parameter declaration options and
+    /// declare a parameter as `Optional`, `Mandatory` or `ReadOnly`.
+    ///
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, ParameterRange, RclrsError};
+    /// let context = Context::new([])?;
+    /// let node = rclrs::create_node(&context, "domain_id_node")?;
+    /// // Set it to a range of 0-100, with a step of 2
```

Comment:
Isn't the range `-100 - 100`?

Reply:
Nice find! Fixed together with all the other nits https://github.com/ros2-rust/ros2_rust/pull/332/commits/25b1bb8b99ee7bd99045df48e575bc2cf4a392a9